react/week2/new-todo/src/App.js

```diff
@@ -0,0 +1,29 @@
+import "./App.css";
+import Title from "./Title";
+import Todolist from "./Todolist";
+
+const todos = [
+  {
+    id: 1,
+    description: "Get out of bed",
+  },
+  {
+    id: 2,
+    description: "Brush teeth",
+  },
+  {
+    id: 3,
+    description: "Eat breakfast",
+  },
+];
+
```

Comment:
For better practice , have a different file only for todos data. We usually do not populate app.js.